app/src/main/java/org/openimis/imispolicies/domain/utils/PhotoUtils.java

```diff
@@ -0,0 +1,35 @@
+package org.openimis.imispolicies.domain.utils;
+
+import android.util.Base64;
+
+import androidx.annotation.Nullable;
+
+public class PhotoUtils {
+
+    private PhotoUtils() throws IllegalAccessException {
+        throw new IllegalAccessException("This constructor should not be accessed");
+    }
+
+    @Nullable
+    public static String getPhotoPath(@Nullable String folder, @Nullable String fileName) {
+        if (fileName != null) {
+            if (folder != null) {
+                folder = folder.replace('\\', '/');
+                if (!folder.endsWith("/")) {
+                    folder += "/";
+                }
+                return folder + fileName;
+            }
+            return fileName;
+        }
+        return null;
```

Comment:
```suggestion
```suggestion
        if (fileName == null) return null;
        if (folder == null) return fileName);
        
        return Paths.get(folder.replace('\\', '/'), fileName).toString();
```
```

Reply:
Paths requires Java 7 which was introduced in Android O (26)

Reply:
thanks for the info